javascript/javascript2/week2/doublingOfNumbers.js

```diff
@@ -0,0 +1,13 @@
+let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+let isOdd = (numbers) => isOdd(numbers) === 'NaN' ? isOdd(numbers) : !isOdd(numbers);
+let oddNumbers = numbers.filter(number => {
+  return number % 2 !== 0;
+});
+
+let doubled = oddNumbers.map(function (numbers){ 
```

Comment:
Try to be consistent with if you're using function expressions or arrow functions.

Also, the argument here would be better names as just `number` since it's only one.

Reply:
ok thanks i will do that